Octokit/Models/Response/RepositoryInvitation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    public enum InvitationPermissionType
```

Comment:
Ah indeed you are right :grinning: the List Invite and Edit Invite calls do have this admin/write/read options.  But the Add and Invite methods need the Admin/Push/Pull.  So i guess you should keep what you have here, but additionally add a `CollaboratorRequest` class that uses the `RepositoryPermission` enum, and use that on the `Invite()` method